howto/monitor/monitor-status.md

```diff
@@ -41,13 +41,13 @@ Complete the following steps to deploy Anbox Cloud with the reference monitoring
      - ['nrpe', 'nagios']
    machines:
      '0':
-       series: bionic
+       series: jammy
```

Comment:
Do the updates in this file make sense?

Reply:
This becomes now even trickier as nagios must stay on bionic. It doesn't support focal or jammy.

Can you leave this one as is for now and I will do a PR next week to clean this up?

Reply:
I dropped the commit.